### PR TITLE
Update templates.md

### PR DESCRIPTION
**Pull request by vseshu**
Adding text based on knowledge I learned from the user forum about how to apply a new default templateVM to sys-usb for devices that are dependant upon sys-usb running for keyboards and mice, etc. I found this to be a very useful set of scripts to run.

### CODE REVIEW

File: user/managing-os/templates.md [hunk 1]
```diff
@@ -137,6 +137,16 @@ When you install a new template or upgrade a clone of a template, it is recommen
         [user@dom0 ~]$ qvm-features new-template-dvm appmenus-dispvm 1
         [user@dom0 ~]$ qubes-prefs default-dispvm new-template-dvm
 
+4. Updating the template for sys-usb if peripheral devices are dependent upon the VM
+    
+    If you are running Qubes on a Desktop or other device where the peripheral devices such as keyboard / mouse / trackpad are dependant upon the sys-usb appVM then updating the template is a challenge. In this situation, you can use the following commands in a dom0 terminal window to update the templateVM.
+    
+        $ qvm-shutdown --wait sys-usb; \
+        $ qvm-prefs sys-usb template fedora-31; \
+        $ qvm-start sys-usb
```
Comment (marmarek) [inline user/managing-os/templates.md:146]: It is important for this to be on one command - putting `$` at each line will break it. But also adding space after `\` (which is easily done by mistake). Better list all three in one line.

Comment (vseshu) [reply user/managing-os/templates.md:146]: aahh, good point. ok so the preferred command should be
$ qvm-shutdown --wait sys-usb | qvm-prefs sys-usb template fedora-31 | qvm-start sys-usb 

I'm a bit of a newbie to this, so did I get the command line correct?

Comment (marmarek) [reply user/managing-os/templates.md:146]: Semicolons like you did initially are fine.

Comment (vseshu) [reply user/managing-os/templates.md:146]: Thanks!

**Pull request merged**